tools/wpt/testfiles.py

```diff
@@ -383,7 +383,7 @@ def get_revish(**kwargs):
     revish = kwargs.get("revish")
     if revish is None:
         revish = u"%s..HEAD" % branch_point()
-    return ensure_text(revish).strip()
```

Comment:
I'm not familiar enough with this file but I guess there's a reason you left the other six instances?

Reply:
Yes, I'm removing them in a coming "Rely on os.path.* returning strings (not bytes)" PR which depends on this. I thought it would be easier to review in themes rather than file-by-file.

Reply:
Now up at https://github.com/web-platform-tests/wpt/pull/28926.